skrub/_minhash_encoder.py

```diff
@@ -341,7 +339,7 @@ def transform(self, X: ArrayLike) -> NDArray:
         # Handle missing values
         missing_mask = (
             ~(X == X)  # Find np.nan
-            | (X == None)  # Find None. Note: `X is None` doesn't work.
+            | (X is None)  # Find None. Note: `X is None` doesn't work.
```

Comment:
This raises an error and must be reverted. See also the inline comment on the right.

Reply:
OMG! I am sorry! I will revert it! (it just looks odd hehe)

Reply:
reverted in https://github.com/skrub-data/skrub/pull/773/commits/795ba3546f26420cbcb943783fbe82452af242ca

Reply:
Added in https://github.com/skrub-data/skrub/pull/773/commits/5a52b2011b50af638bc20a31d1cdbcf92d64b456

Reply:
> OMG! I am sorry! I will revert it! (it just looks odd hehe)

Yes, there is room for improvement. Feel free to suggest enhancement in subsequent PRs or open an issue if you see fit :)

Reply:
ok! I will do so 👍